src/app-bar.jsx

```diff
@@ -127,6 +159,7 @@ const AppBar = React.createClass({
       showMenuIconButton: true,
       title: '',
       zDepth: 1,
+      position: 'fixed',
```

Comment:
Hmmm you do have a point, @oliviertassinari What do you think? maybe just put a note in the release. I guess there are more people using AppCanvas than AppBar alone.